sdk/core/Azure.Core/tests/HttpPipelineFunctionalTests.cs

```diff
@@ -628,7 +628,7 @@ public async Task SendMultipartformData()
         }
 
         [Test]
-        public async Task HandlesRedirects()
+        public async Task HandlesRedirects([Values(true, false)] bool allowRedirects)
```

Comment:
Please use TestCase to match other tests in this file.

Reply:
It's not the only one 😄